core/src/UserSettingsEditor.html

```diff
@@ -152,8 +152,8 @@
                   <input
                     type="text"
                     class="fd-input fd-input-group__input"
-                    placeholder="{$getTranslation(storedUserSettingData[userSettingGroup[0]][key])}"
                     data-testid="lui-us-input{i}"
+                    placeholder="{GenericHelpers.isObject(storedUserSettingData[userSettingGroup[0]][key])?$getTranslation(Object.values(storedUserSettingData[userSettingGroup[0]][key])[0]):$getTranslation(storedUserSettingData[userSettingGroup[0]][key])}"
```

Comment:
This line seems quite long.
I'm thinking of two options here: 
- Do you think it would make sense to extract this into a **GenericHelpers. getUserSettingsFromhmhmhm...........**
- Maybe, that's too much but, what about breaking this line up into the condition and the two options, it would maybe be bit better than the whole line. Sth similar or if you have a better split in mind:
```
// I didn't test this
{GenericHelpers.isObject(storedUserSettingData[userSettingGroup[0]][key]) ? 
        $getTranslation(Object.values(storedUserSettingData[userSettingGroup[0]][key])[0])
        :$getTranslation(storedUserSettingData[userSettingGroup[0]][key])} 
```

I checked and **Prettier** is not able to cut long lines all the time when the line is too long, automatically. Maybe because it is just a string. So, I think Prettier won't complain if you split this to 2-3 lines.